packages/iris-grid/src/IrisGridTableModel.js

```diff
@@ -857,14 +861,19 @@ class IrisGridTableModel extends IrisGridModel {
   }
 
   get formatColumns() {
-    return this.formatColumnList;
+    return this.formatColumnList ?? [];
```

Comment:
Should reference a const object here so it's not a new object every time if the formatColumnList is null